lib/taskana-core/src/main/java/pro/taskana/task/internal/models/TaskSummaryImpl.java

```diff
@@ -238,6 +242,18 @@ public void setPriority(int priority) {
     this.priority = priority;
   }
 
+  @Override
+  public int getManualPriority() {
+    return manualPriority;
+  }
+
+  public void setManualPriority(int manualPriority) {
+    this.manualPriority = manualPriority;
+    if (manualPriority >= 0) {
+      this.priority = manualPriority;
+    }
```

Comment:
```suggestion
    if (isManualPriorityActive()) {
      this.priority = manualPriority;
    }
```

Does this make this better? What do you think?